radixdlt/src/integration/java/com/radixdlt/consensus/simulation/checks/LivenessCheck.java

```diff
@@ -15,36 +15,39 @@
  * language governing permissions and limitations under the License.
  */
 
-package com.radixdlt.consensus.checks;
+package com.radixdlt.consensus.simulation.checks;
 
 import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
 
-import com.radixdlt.consensus.BFTCheck;
-import com.radixdlt.consensus.BFTTestNetwork;
+import com.radixdlt.consensus.simulation.BFTCheck;
+import com.radixdlt.consensus.simulation.BFTSimulation;
 import com.radixdlt.consensus.QuorumCertificate;
 import com.radixdlt.consensus.VertexStore;
 import com.radixdlt.consensus.View;
+import io.reactivex.rxjava3.core.Completable;
 import io.reactivex.rxjava3.core.Observable;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicReference;
 import org.assertj.core.api.Condition;
 
 /**
- * Check that the network is making strong progress by ensuring that new QCs are
- * being created every round.
+ * Check that the network is making progress by ensuring that new QCs are
+ * progressively increasing in view number.
  */
-public class StrongLivenessCheck implements BFTCheck {
-	@Override
-	public Observable<Object> check(BFTTestNetwork network) {
-		// there should be a new highest QC every once in a while to ensure progress
-		// the minimum latency per round is determined using the network latency
-		// a round can consist of 6 * max_transmission_time
-		double trips = 6.0;
-		int maxLatencyPerRound = (int) (network.getMaximumNetworkLatency() * trips);
+public class LivenessCheck implements BFTCheck {
+	private final long time;
+	private final TimeUnit timeUnit;
+
+	public LivenessCheck(long time, TimeUnit timeUnit) {
```

Comment:
As noted earlier, the meaning of `time` is not evident here until you look at the implementation of `check`.

Reply:
As noted elsewhere, I think this should be "duration" instead of "time".